src/Playwright/Artifact.cs

```diff
@@ -6,6 +6,8 @@
 using Microsoft.Playwright.Transport.Channels;
 using Microsoft.Playwright.Transport.Protocol;
 
+#pragma warning disable CS1591, SA1600
```

Comment:
This should go into https://github.com/microsoft/playwright-sharp/blob/main/src/.editorconfig#L178

Reply:
Let me try this...